app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -121,6 +121,25 @@ fun ApprovalButton(
     }
 }
 
+@Composable
+fun InfoButton(modifier: Modifier = Modifier, isEnabled: Boolean,
```

Comment:
Since this button is always enabled we don't need to expose "isEnabled" and can hardcode the value for the depth button.

Reply:
Addressed..!!